### DISCUSSION TITLE
Is composer still supported ?

### DISCUSSION THREAD

**Discussion by warlof** [tabler/tabler#992] [Q&A]
Hi everyone,

I would use tabler as my project new template and benefits of all its shiny well designed UI assets.
Since it's a PHP project, I mostly relaying on composer.

I noticed tabler was published on packagist repository, however, published version seems to be outdated ou unsynched.

So question is : Is composer still maintain or not ?

**Comment by kevinpapst** [comment]
@codecalm If you want to, you can setup auto-update, composer will then fetch your releases automatically. 
See https://packagist.org/about#how-to-update-packages - it's just a matter of installing the Composer Github app.

@warlof I am working on a Symfony integration, just in case your project is based on Symfony, [you could have a look](https://github.com/kevinpapst/TablerBundle).

**Reply by warlof** [reply]
My project is using Laravel unfortunately. There is an existing wrapper, but designed for Laravel 5 only 😋
